src/libraries/System.Text.RegularExpressions/src/System/Text/RegularExpressions/RegexCompiler.cs

```diff
@@ -5139,6 +5147,41 @@ private void EmitMatchCharacterClass(string charClass)
                     Call(s_charIsSymbolMethod);
                     NegateIf(charClass == RegexCharClass.NotSymbolClass);
                     return;
+
+                case RegexCharClass.AsciiLetterClass:
+                case RegexCharClass.NotAsciiLetterClass:
+                    // char.IsAsciiLetter(ch)
+                    Call(s_charIsAsciiLetterMethod);
```

Comment:
I assume there's no concern for the output of this to be an older runtime where these members aren't available.

Reply:
Unless someone created a frankenbuild with System.Text.RegularExpressions from .NET 7 and tried to use it with a pre-.NET 7 runtime, no.